### DISCUSSION TITLE
SocketIpDriverCfg is substituted by IpCfg?

### DISCUSSION THREAD

**Discussion by SMorettini** [nasa/fprime#1891] [General]
I notice in `Fprime/config` there are two very similar files: `SocketIpDriverCfg.hpp` is substituted by `IpCfg.hpp`. The first one is never included inside Fprime. It is a duplicate? Should we remove it?

**Comment by LeStarch** [comment]
Yes the first one should be removed and the following comment reference should be updated:
https://github.com/nasa/fprime/blob/7bf79db61b1b63d60a624328dc909ee62869e8ba/Drv/Ip/IpSocket.hpp#L102

**Reply by SMorettini** [reply]
https://github.com/nasa/fprime/issues/1895
